model_metadata.py ran successfully